utils/check-service

```diff
@@ -75,6 +75,13 @@ declare -A status=(
     [failing]=2
 )
 
+fid2id() {
```

Comment:
[optional] Some documentation and an example would be useful.

Reply:
Agree. Will add.